app/assets/stylesheets/description_box.scss

```diff
@@ -0,0 +1,8 @@
+.description {
+    border: 1px #333;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces